kibana-reports/server/routes/utils/converters/backendToUi.ts

```diff
@@ -222,12 +222,15 @@ const getUiQueryUrl = (
   const timeFrom = moment(beginTimeMs).toISOString();
   const timeTo = moment(endTimeMs).toISOString();
   let queryUrl = `${baseUrl}?_g=(time:(from:'${timeFrom}',to:'${timeTo}'))`;
-  if (tenant) {
-    if (tenant === '__user__') {
+  if (tenant !== undefined) {
+    if (tenant === '') {
+      tenant = 'global';
```

Comment:
what if the user creates a new tenant with name "global"?

Reply:
then after this if else check, `tenant` variable will be assigned with `global`. It will be added to the url without any issue. Same for global tenant. they follow the same format in the url `?security_tenant=global` or `?security_tenant=<customized_tenant_name>`. From the actual behavior point, it's out of our control. Not sure seucirty plugin has a way to distinguish that.